src/components/swap/ConfirmSwapModal.tsx

```diff
@@ -241,8 +241,9 @@ export default function ConfirmSwapModal({
         hideStepIndicators={pendingModalSteps.length === 1}
         steps={pendingModalSteps}
         currentStep={confirmModalState}
-        approvalCurrency={trade?.inputAmount?.currency}
-        txHash={txHash}
+        trade={trade}
+        swapHash={txHash}
```

Comment:
nit: variable naming
```suggestion
        swapTxHash={txHash}
```